CHANGELOG.md

```diff
@@ -6,16 +6,20 @@ Breaking changes result in a different major. UI changes that might break custom
 
 ## [next-version]
 
+## [0.14.0-rc.2]
+
 ### Changed
 
 - Public: Document and face captures will be returned by callbacks as File or Blob instead of base64.
+- Internal: Callbacks are now linked to the flow rather than the Redux store.
```

Comment:
this is helpful for us, but there might be something helpful for the use. We have now changed the behaviour of the `onComplete` callback, right?